src/scheduler.rs

```diff
@@ -553,30 +553,10 @@ async fn fetch_idle(
     // be able to scan all folders before time is up if there are many of them.
     if folder_config == Config::ConfiguredInboxFolder {
         // Only scan on the Inbox thread in order to prevent parallel scans, which might lead to duplicate messages
-        match connection.scan_folders(ctx).await.context("scan_folders") {
-            Err(err) => {
-                // Don't reconnect, if there is a problem with the connection we will realize this when IDLEing
-                // but maybe just one folder can't be selected or something
-                warn!(ctx, "{:#}", err);
-            }
-            Ok(true) => {
-                // Fetch the watched folder again in case scanning other folder moved messages
-                // there.
-                //
-                // In most cases this will select the watched folder and return because there are
-                // no new messages. We want to select the watched folder anyway before going IDLE
-                // there, so this does not take additional protocol round-trip.
-                if let Err(err) = connection
-                    .fetch_move_delete(ctx, &watch_folder, folder_meaning)
-                    .await
-                    .context("fetch_move_delete after scan_folders")
-                {
-                    connection.trigger_reconnect(ctx);
-                    warn!(ctx, "{:#}", err);
-                    return InterruptInfo::new(false);
-                }
```

Comment:
@link2xt Did you remove this by accident, or don't we need it anymore?

Don't we need to fetch Inbox again after scanning? (IIUC this is about the Spam folder, because I can't think of another folder from which we would want to move messages to the Inbox).

Or is it just that the added complexity isn't worth the speedup? (IIUC this was about performance, not correctness, because when going IDLE the server would notify us about new messages)